Code/bin/main.py

```diff
@@ -23,31 +23,41 @@ def loss(original:Sequence[Label],applied:Sequence[Label], miss_w=1.0,extra_w=1.
     return miss_w*len(misses) + extra_w*len(extras)
 
 
-def run(plays:Sequence[tuple[DataFrame,str]]):
+def label_plays(plays:Sequence[tuple[DataFrame,str]]):
+    ## TODO FIXME: use actual labeling function according to vectors.
+    ## Instead of `parse_fragment()`
     for df,name in plays:
         # Label it again
         df['Labels'] = df.apply(
         lambda row: convert_labels(parse_fragment(row.Fragment)),
         axis = 1
         )
         print(f"\t > Applied labels on {name}")
+
+            
+def check_labeling(plays:Sequence[tuple[DataFrame,str]]):
+    ## TODO FIXME: use actual loss function.
```

Comment:
@Erezito 
I made the wrapper for calculating loss
need to figure out how do calculate loss given two lists (hand labeled vs script labeled)
I used a naive method for now of counting the mismatches (pretty simple you can have a look).